src/main/java/eu/bitwalker/useragentutils/Browser.java

```diff
@@ -373,7 +373,7 @@ public Version getVersion(String userAgentString) {
 		Pattern pattern = this.getVersionRegEx();
 		if (userAgentString != null && pattern != null) {
 			Matcher matcher = pattern.matcher(userAgentString);
-			if (matcher.find()) {
+			if (matcher.find() && matcher.groupCount() > 1) {
```

Comment:
should this be checking if `matcher.groupCount() >= 2` here to ensure that majorVersion isn't null?